src/app/forms/components/number-input-with-suffix/number-input-with-suffix.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="govuk-form-group">
+  <h1 class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--m" for="{{ name }}">{{ label }}</label>
+  </h1>
+
+  <app-field-error-message [error]="error" [name]="name"></app-field-error-message>
+
+  <div *ngIf="isEditing; else view" class="govuk-input__wrapper">
+    <input
+      class="govuk-input govuk-input--width-3"
```

Comment:
width class could be an ngClass with an input size modifier.

Reply:
Working on another section now. Will implement this. Keep an eye out for my next PR 😁